packages/tools/generator-widget/generators/app/lib/prompttexts.js

```diff
@@ -18,28 +18,52 @@ function promptWidgetProperties(mxProjectDir, widgetName) {
             type: "input",
             name: "description",
             message: "Enter a description for your widget",
-            default: "My widget description"
+            default: "My widget description",
+            validate: input => {
+                if (/^[\w\s!?.'"-]+$/gi.test(input)) {
```

Comment:
Wouldn't it be far more user-friendly to simply escape the description (and the other properties) in the generated output? I am not aware of any technical reason why a `package.json` wouldn't allow doublequotes, as long as they are properly escaped.

On a side note: why add the `i` flag on regexes which do not contain any text literals? :)

Reply:
> On a side note: why add the i flag on regexes which do not contain any text literals? :)

Ooops, this is C&P

Reply:
Just to note, I added quote escaping for description, but not included it for other fields, because I'm not sure about them. I think fields like `name`, `author` and `copyright` should be more simple and we should prevent users from adding quotes inside. But if you think this is bad UX I can add escaping for other fields.

Reply:
@tamelbrun

Reply:
The `name` already has a restriction for technical reasons, which we should obviously keep. But I don't think we should add any unnecessary restrictions, unless something is being actively discouraged in the `package.json` format. For example, if you want the author's name to be `Clark "Superman" Kent`, why would we disallow that?